### ISSUE TITLE
use-component-selector does not accept `` strings

### ISSUE THREAD

**Issue by michael-letcher** [angular-eslint/angular-eslint#446]
I get an error if you use backticks as the selector

Config: `"extends": ["plugin:jest/recommended"]`

This fails
```ts
@Component({
    selector: `test-host-component`, // <= Issue here
    template: `TEMPLATE STUFF`,
})
```

This passes
```ts
@Component({
    selector: 'test-host-component',
    template: `TEMPLATE STUFF`,
})
```

**Comment by JamesHenry** [comment]
@michael-letcher Please help me to help you, and try and put yourself in my shoes when only receiving this small amount of context. There is not enough information here to reproduce your issue, and it is not clear what you are referring to when you say "an error" and "issue here".

- Do you mean an exception was thrown? If so what exception? What is the stack trace?
- In general what tooling is involved? Is this `ng lint`, VSCode, or both?
- What is your full eslint config?

I provided issues templates for precisely this reason, they try and guide users to providing this information up front.

Finally, I have wired up a Component in a project to use your code example above, and extend from that `eslint-plugin-jest` shareable config and I do not encounter any issues via `ng lint` or in VSCode.

Please feel to reopen once you can provide more information, naturally a reproduction repo is always best.
